l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -137,11 +137,7 @@ def open(self, mode: str = "r", cached: bool = True, cache_size_bytes: int = int
         self.frames = self.root[FRAME_ARRAY_KEY]
         self.agents = self.root[AGENT_ARRAY_KEY]
         self.scenes = self.root[SCENE_ARRAY_KEY]
-        try:
-            self.tl_faces = self.root[TL_FACE_ARRAY_KEY]
-        except KeyError:
-            print(f"{TL_FACE_ARRAY_KEY} not found in {self.path}! Traffic lights will be disabled")
-            self.tl_faces = np.empty((0,), dtype=TL_FACE_DTYPE)
+        self.tl_faces = self.root[TL_FACE_ARRAY_KEY]
```

Comment:
So we just crash? Wasn't the previous version better?

Reply:
I agree; It was better for retro-compatibility, but the thing is that we want to completely move to traffic light datasets only. So it makes sense to hard crash here imo

Reply:
I have restored this with a stronger warning. For now we will still support loading a zarr without traffic lights but not in the rest of the workflow (e.g. you won't be able to use it in an EgoDataset). In this way we can still inspect raw legacy data if we want to